neutral_generation/smart_convert.py

```diff
@@ -4,8 +4,9 @@
 import spacy
```

Comment:
This file is getting really big and bulky. Let's try and add tests for each function here in a new PR. That's good practice and would also enable us to identify bugs easily. Checking this PR in and then adding a new one for tests later is fine.

Reply:
You got it.